commands/server.go

```diff
@@ -48,6 +48,9 @@ type serverCmd struct {
 	// Can be used to stop the server. Useful in tests
 	stop <-chan bool
 
+	// Sent to when the server is ready for connections. Used for testing
+	ready chan struct{}
```

Comment:
I have no strong opinion about what to use, but the type should be the same for both of `stop` and `ready`.